### PR TITLE
RFC: Add image manifest size to report.toml

### PR DESCRIPTION
**Pull request by djoyahoy**
[Readable](https://github.com/djoyahoy/rfcs/blob/main/text/0000-report-toml-image-manifest-size.md)

### CODE REVIEW

**Review by jromero — APPROVED** [review]

**Review by ekcasey — APPROVED** [review]

File: text/0000-report-toml-image-manifest-size.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+# Alternatives
+[alternatives]: #alternatives
+
+N/A
+
+# Prior Art
+[prior-art]: #prior-art
+
+This would be the exact same size as reported by docker push.
```
Comment (jkutner) [inline text/0000-report-toml-image-manifest-size.md:89]: does docker store the value anywhere, or does it just appear in the output of `docker push`?

File: text/0000-report-toml-image-manifest-size.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+# Unresolved Questions
+[unresolved-questions]: #unresolved-questions
+
+The name of the field is subject to change.
+
+# Spec. Changes (OPTIONAL)
+[spec-changes]: #spec-changes
+
+This RFC would add a new field to the `report.toml` file.
```
Comment (jkutner) [inline text/0000-report-toml-image-manifest-size.md:99]: can you show the diff to [report.toml](https://github.com/buildpacks/spec/blob/main/platform.md#reporttoml-toml) in the spec here?

File: text/0000-report-toml-image-manifest-size.md (hunk 1 above)
Comment (djoyahoy) [reply text/0000-report-toml-image-manifest-size.md:89]: As far as I know, this value is not stored anywhere.

**Review by nebhale — APPROVED** [review]

**Review by hone — APPROVED** [review]

**Review by jkutner — APPROVED** [review]

**Comment by nebhale** [comment]
Final Comment Period with merge disposition, closing on 16 December, 2020.

**Pull request merged**